src/components/Select/Select.tsx

```diff
@@ -38,7 +38,7 @@ export interface BaseSelectProps {
      */
     disabled?: boolean;
     /**
-     * This is to set the searchable property of the dropdown. 
+     * This is to set the searchable property of the dropdown.
      * If set to true, the user can type the value to search from the available options.
      */
     isSearchable?: boolean;
```

Comment:
nit: I know you didn't change this line, but I just wanted to say that I really dislike boolean properties that start with `is`. If we imagine this property was a string, for example, we wouldn't call it `getSearchable`, so I believe we should be calling boolean properties by their name. In this case: `searchable`.

Reply:
will write a story to complete this for all components